app/src/main/java/com/kickstarter/mock/services/MockApolloClient.kt

```diff
@@ -56,11 +57,7 @@ open class MockApolloClient : ApolloClientType {
                 "12345")))
     }
 
-    override fun updateBacking(backing: Backing, amount: String, locationId: String?, reward: Reward?): Observable<Boolean> {
-        return Observable.just(true)
-    }
-
-    override fun updateBackingPayment(backing: Backing, paymentSourceId: String): Observable<Boolean> {
+    override fun updateBacking(updateBacking: UpdateBacking): Observable<Boolean> {
```

Comment:
thank you for not adding one more argument and instead converting to single object.